lib/launcher.ts

```diff
@@ -93,7 +92,7 @@ let taskResults_ = new TaskResults();
  * @param {string=} configFile
  * @param {Object=} additionalConfig
  */
-let initFn = function(configFile: string, additionalConfig: Config) {
+let initFn = async function(configFile: string, additionalConfig: Config) {
   let configParser = new ConfigParser();
   if (configFile) {
```

Comment:
do we need it still here with `TS`? Probably, not.

Reply:
I am not sure. Could you clarify?